### ISSUE TITLE
The App loads but doesn't show any content

### ISSUE THREAD

**Issue by WhatsUPCrypto** [popcorn-official/popcorn-android#754]
## Describe the bug

On launching the app, it loads, after loading doesn't show anything, the app is empty. No titles, no movies or anything else.

## Expected behavior

Seeing the movies and other content after all launch.

## Smartphone (please complete the following information):

 - Device: Motorola one vision
 - OS: Android 10
 - Browser chrome 
 - Version 90.0.4430.82

**Comment by ivan1986** [comment]
Main api is down, only ru-tk server online now - it's fork with localization - you may download android build here - http://popcorn-ru.tk/build/
it's build of fork - https://github.com/popcorn-time-ru/popcorn-android - support localization, not crash if exist custom fields
adding custom api servers in plans

**Comment by Jany-M** [comment]
> Main api is down, only ru-tk server online now - it's fork with localization - you may download android build here - http://popcorn-ru.tk/build/
> it's build of fork - https://github.com/popcorn-time-ru/popcorn-android - support localization, not crash if exist custom fields
> adding custom api servers in plans

Unfortunately their Android arm version is bugged (on movie tap it brings you back to movies tab view) and TV version has movie progress bar stuck, won't go away when watching... :(

**Comment by ivan1986** [comment]
@Jany-M yes, find error in tv - fixed, should work, but i can not normal testing tv version, mobile works

**Comment by Jany-M** [comment]
> @Jany-M yes, find error in tv - fixed, should work, but i can not normal testing tv version, mobile works

Great! Does auto update work on the Android and TV version? or do I need to uninstall and reinstall?
Issues are not open on that repo, I can post feedback there in the case of other issues.

**Comment by ivan1986** [comment]
@Jany-M no, autoupdate not working, now i just make builds and put it's on site
